test/acceptance/support/azure-client.js

```diff
@@ -0,0 +1,47 @@
+const { BlobServiceClient } = require('@azure/storage-blob')
+const { ServiceBusClient } = require('@azure/service-bus')
+const config = require('./config')
+
+let blobServiceClient = BlobServiceClient.fromConnectionString(`${config.connectionStr}`)
+let container = blobServiceClient.getContainerClient(`${config.container}`)
+
+const uploadFile = async (filename, filepath) => {
+  const blob = container.getBlockBlobClient(`${config.inboundFolder}/${filename}`)
+  await blob.uploadFile(filepath)
+}
+
+const receiveMessage = async () => {
```

Comment:
Suggest pluralising this to make the behaviour clearer.  It's returning an array of messages rather than a single message.  So `receiveMessages` seems more accurate.